scss/components/_overlay.scss

```diff
@@ -21,3 +21,17 @@
   }
 }
 
+.overlay--neutral {
+  background-color: transparent;
+
+  &.is-loading {
+    background-image: url('../img/loading--primary.gif');
+    background-repeat: no-repeat;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat